features/managing_messages/send_admin_message.feature

```diff
@@ -0,0 +1,19 @@
+@message_form
+Feature: Send message
+  In order to contact the registered customer
+  As an Administrator
+  I want to have an appropriate form on the administrator panel
+
+  Background:
+    Given the store operates on a single channel in "United States"
+    And there is a customer "sylius@mangoweb.cz" that send a contact form
```

Comment:
`And there is a customer "sylius@mangoweb.cz" that send a contact form` => `And there is a customer "sylius@mangoweb.eu" that submits a contact form`